taxi/views.py

```diff
@@ -80,10 +107,31 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
 
 
 class DriverListView(LoginRequiredMixin, generic.ListView):
-    model = Driver
+    model = get_user_model()
```

Comment:
Ensure that the user model returned by `get_user_model()` includes all necessary fields, such as `license_number`, to be compatible with the forms and views that use it.